app/Jobs/ApplicationDeploymentJob.php

```
@@ -1342,22 +1454,26 @@ private function create_workdir()
 
     private function prepare_builder_image()
     {
+        $this->checkForCancellation();
         $settings = instanceSettings();
         $helperImage = config('constants.coolify.helper_image');
         $helperImage = "{$helperImage}:{$settings->helper_version}";
         // Get user home directory
         $this->serverUserHomeDir = instant_remote_process(['echo $HOME'], $this->server);
         $this->dockerConfigFileExists = instant_remote_process(["test -f {$this->serverUserHomeDir}/.docker/config.json && echo 'OK' || echo 'NOK'"], $this->server);
+
+        $env_flags = $this->generate_docker_env_flags_for_secrets();
```

PHPMD (2.15.0)

The variable $env_flags is not named in camelCase. (undefined)

(CamelCaseVariableName)